Cargo.toml

```diff
@@ -13,6 +13,7 @@ byteorder = "=1.2.1"
 epoll = "=4.0.1"
 libc = ">=0.2.39"
 log = "*"
+vmm-sys-util = { git = "https://github.com/liujing2/vmm-sys-util" }
```

Comment:
This should probably point to a crate under the rust-vmm org sooner or later.

Reply:
Yap, we are using the fork for now because the vmm-sys-util PR is not yet merged. Once https://github.com/rust-vmm/vmm-sys-util/pull/1 is merged we will move to use the official vmm-sys-utils repository as a dependency.